src/app/settings.py

```diff
@@ -347,3 +355,20 @@ def show_toolbar(request):
 USE_FIFO = os.environ.get("USE_FIFO", "True")
 # Override default boto3 url for SQS
 ENDPOINT_URL = None if ENVIRONMENT in ("dev", "prod") else "http://sqs:9324"
+# AWS S3 bucket for public files
```

Comment:
I reorganized buckets into one section above in another branch so duplicates should be removed